integration_test/driver.cpp

```diff
@@ -5,7 +5,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "Vmain.h"
+#include "Vtop.h"
```

Comment:
Given that you have code that needs to be built, it might make sense to move circt-rtl-sim.py (and the .cpp and .sv file) into circt/tools/circt-rtl-sim/.  This would make sure the .o file for the .cpp file is always built before tests are run, simplifying the python file.

Reply:
This would also be a more logical place to put testing utilities like this than in integration-test itself.

Reply:
This gets compiled along with the code which Verilator produces by Verilator so we don't compile it as part of the cmake build. I agree about the location.